<?xml version="1.0"?>

<launch>
	<node name="youbot_battery_control" pkg="youbot_battery_control" type="youbot_battery_control">
		<param name="charge_voltage" type="double" value="12.0"/>
		<param name="critical_cell_voltage" type="double" value="11.0"/>
		<param name="waning_cell_voltage" type="double" value="11.5"/>
		<param name="no_info_sleep_time" type="double" value="3.0"/>
		<param name="time_to_critical_state" type="double" value="1.0"/>
		<param name="time_to_warning_state" type="double" value="1.0"/>
		<param name="time_to_good_state" type="double" value="1.0"/>
		<param name="time_to_not_power_state" type="double" value="0.1"/>
		<param name="warning_command_interval" type="double" value="60.0"/>
		<param name="critical_command_interval" type="double" value="0.0"/>
		<param name="warning_command" type="str" value="/usr/local/etc/beep_sounds/ring"/>
		<param name="critical_command" type="str" value="/usr/local/etc/beep_sounds/alarm"/>
	</node>
</launch>


